samples/tfm_integration/tfm_secure_partition/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cmake_minimum_required(VERSION 3.13.1)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+
+configure_file(
+  ${CMAKE_CURRENT_LIST_DIR}/dummy_partition/tfm_manifest_list.yaml
```

Comment:
Is there any convention with how to name a file that is configured in this way? IMO it would be good for them to have different names so that it is easy to see which is which when grepping or searching for files.

Reply:
Common to append `.in` to the name. Renamed.